src/notification.cpp

```diff
@@ -391,6 +398,11 @@ class NotificationPrivate : public NotificationData
                 "name": "ignore",
                 "displayName": "Ignore the problem",
                 "icon": "icon-s-ignore",
+                "reply" : {
+                    "label": "Please select:",
+                    "editable": true,
+                    "choices": [ "Yes", "No", "Maybe" ]
+                },
```

Comment:
Clean definition, but then again the d-bus callback is defined without grouping. So either we are consistent or clean here :) 

For minor adjustment could leave the colon out of the label, depends on UI style whether it makes sense.